cmd/zeno/zeno.go

```diff
@@ -22,7 +22,7 @@ import (
 	"github.com/getlantern/zenodb/common"
 	"github.com/getlantern/zenodb/planner"
 	"github.com/getlantern/zenodb/rpc"
-	"github.com/getlantern/zenodb/rpc/server"
+	rpcserver "github.com/getlantern/zenodb/rpc/server"
```

Comment:
I don't see this updated anywhere else.  If it's not being used, shouldn't Go complain about this import?  And if it's being used, shouldn't this name change break usages?

Reply:
Yeah that's an auto fix that goimports I think started doing. rpcserver is used on line 291, but before it was just implicit that rpc/server happened to use the namespace rpcserver (not sure exactly what's happening underneath to do that). so this is just clearer.

Reply:
Oh, I see it was the package name already: https://github.com/getlantern/zenodb/blob/master/rpc/server/rpc_server.go#L1.  Now I remember that the bit about the package names matching the last path element is just a convention in golang, not enforced by the compiler.

Reply:
@myleshorton Yeah, I changed that to just run `gofmt` because `goimports` actually sometimes introduces breaking changes for me.

Reply:
Yeah I generally like the changes `goimports` makes so stick with it.